lib/Doctrine/Common/Cache/WinCacheCache.php

```diff
@@ -83,6 +83,20 @@ protected function doFetchMultiple(array $keys)
     /**
      * {@inheritdoc}
      */
+    protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
+    {
+        $result = wincache_ucache_set($keysAndValues, null, $lifetime);
+
+        if ($result === false || count($result)) {
```

Comment:
Same as above: rely on `empty()` and don't check for `false` if an array is returned